Formula/arxiv_latex_cleaner.rb

```diff
@@ -0,0 +1,39 @@
+class ArxivLatexCleaner < Formula
+  include Language::Python::Virtualenv
+
+  desc "Clean LaTeX code to submit to arXiv"
+  homepage "https://github.com/google-research/arxiv-latex-cleaner"
+  url "https://files.pythonhosted.org/packages/d7/82/b32e1991ee2a8fd0db2f02ba8e135430aaeed9507e8008d5e9c8beec8eff/arxiv_latex_cleaner-0.1.27.tar.gz"
+  sha256 "f347fc6082417316247dca97bb76f15ac677eb1f5e65f091013086a30cac4eda"
+  license "Apache-2.0"
+  head "https://github.com/google-research/arxiv-latex-cleaner.git", branch: "main"
+
+  depends_on "pillow"
```

Comment:
We need to add an entry to `pypi_formula_mappings.json` to ignore `pillow` resource for the following updates.
Also, `absl_py` depends on `six` so we need to add `depends_on "six"` here (and ignore `six` in `pypi_formula_mappings.json` as well)
```
$ pipgrip arxiv_latex_cleaner
arxiv-latex-cleaner==0.1.27
absl-py==1.0.0
six==1.16.0
pillow==9.1.0
pyyaml==6.0
```

Reply:
Got it. Added `six` in [7a28063](https://github.com/Homebrew/homebrew-core/pull/100405/commits/7a280638b11b1436a4a2cfaf11570a4834ad3996) and `pypi_forumla_mappings.json` in [225c076](https://github.com/Homebrew/homebrew-core/pull/100405/commits/225c076ce4848a6b140b6f732bb634a30255a6a3).